apps/src/templates/sectionAssessments/StudentAssessmentOverviewTable.jsx

```diff
@@ -49,11 +67,15 @@ class StudentAssessmentOverviewTable extends Component {
   };
 
   getCorrectAnswer = (multipleChoiceAnswers) => {
-    return multipleChoiceAnswers.filter(answerObj => {
+    const answersArr = multipleChoiceAnswers.filter(answerObj => {
       return answerObj.isCorrectAnswer;
-    }).map(answerObj => {
+    });
```

Comment:
this call to `.filter` and the call to `.map` below could be simplified to one-line expressions like:
```
const answersArr = multipleChoiceAnswers.filter(answerObj => answerObj.isCorrectAnswer);
const getCorrectAnswers = answersArr.map(answerObj => answerObj.multipleChoiceOption);
```
this definitely isn't a blocker for merging this PR, just letting you know 😄

Reply:
Thank you!